changelog/7005.bugfix.md

```diff
@@ -0,0 +1 @@
+Add encoding ´utf-8´ to write_dot funcion
```

Comment:
As this goes into a big change log file, the following gives a bit more context:
```suggestion
Fixed a bug for plotting trackers with non-ascii texts during interactive training by enforcing utf-8 encoding
```